docs/migration-guides/1.19-1.20.md

```diff
@@ -0,0 +1,82 @@
+# Migrate from 1.19 to 1.20
+
+## Fluent Bit
+
+Kyma 1.20 comes with a redesigned configuration for Fluent Bit. It is redesigned to be up to date with the upstream chart and also be consistent with the rest of the configuration values. The latest configurations can be found [in the chart](https://github.com/kyma-project/kyma/blob/master/resources/logging/charts/fluent-bit/values.yaml#L134). If you have been using overrides for Fluent Bit, compare them with the latest configurations and adapt if necessary. It also now supports many more configuration options for the plugins coming out of the box. For example, you can set the `alias` field for all the input and output plugins to be used as the parent name for the metric. You can read more about it [here](https://docs.fluentbit.io/manual/administration/monitoring#configuring-aliases).  
+
+An example migration looks like the following:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logging-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: logging
+    kyma-project.io/installation: ""
+data:
+  fluent-bit.conf.Output.http.enabled: "true"
+  fluent-bit.conf.Output.http.Match : "*"
+  fluent-bit.backend.http.host: "<Fluentd-endpoint from service key json>"
+  fluent-bit.backend.http.port: "443"
+  fluent-bit.backend.http.http_user: "<Fluentd-username from service key json>"
+  fluent-bit.backend.http.http_passwd: "<Fluentd-password from service key json>"
+  fluent-bit.backend.http.tls: "true"
+  fluent-bit.backend.http.tls_verify : "false"
+```
+
+Old overrides
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logging-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: logging
+    kyma-project.io/installation: ""
+data:
+  fluent-bit.config.outputs.http.enabled: "true"
+  fluent-bit.config.outputs.http.match : "*"
+  fluent-bit.config.outputs.http.host: "<Fluentd-endpoint from service key json>"
+  fluent-bit.config.outputs.http.port: "443"
+  fluent-bit.config.outputs.http.httpUser: "<Fluentd-username from service key json>"
+  fluent-bit.config.outputs.http.httpPasswd: "<Fluentd-password from service key json>"
+  fluent-bit.config.outputs.http.tls.enabled: "true"
+  fluent-bit.config.outputs.http.tls.verify : "false"
+```
+
+New overrides
+
+This configuration can also be passed using the new additional keyword. Additional keyword exists also for inputs, filters, and parsers.
```

Comment:
Does that mean "An additional keyword exists..." or "Additional keywords exist..." ? 
If it's just one additional keyword, we need the indefinite article "an".

Reply:
There are new fields called `additional`. I tried to make it more understandable.